test-result-summary-client/src/TabularView/TabularView.jsx

```diff
@@ -0,0 +1,703 @@
+    onBenchmarkChange = value => {
+        this.setState({benchmarkFilter: value});
+        this.benchmarkFilter(value);
+    }
+    // Date picker format needs to be converted to match the one available in the database entry
+    dateTransform(date, type) {
+        const dateSplit = date.split('/');
+        const jdkDate = dateSplit[2] + ((dateSplit[0].length < 2) ? "0" + dateSplit[0] : dateSplit[0]) + ((dateSplit[1].length < 2) ? "0" + dateSplit[1] : dateSplit[1]);
+
+        if (type === 'test') {
+            this.setState({testJdkDate: jdkDate});
+        } else {
+            this.setState({baselineJdkDate: jdkDate});
+        }
+    }
+    // Based on data returned from database, create relevant columns for platforms
+    generateColumns(platforms) {
+        const newArray = [];
+        let column = {
+        Header: 'Benchmark Name',
+        accessor: 'benchmarkName',
+        Cell: props => <span>{props.value.split(",")[0]} <br/> {props.value.split(",")[1]} <br/> {props.value.split(",")[2]}</span>
+        };
+
+        newArray.push(column);
+
+        for (let i = 0; i < platforms.length; i++) {
+            let platform = platforms[i];
+    	
+            column = {};
+            column.id = platform;
+            column.Header = platform.toUpperCase();
+            column.accessor = d => d.platforms[platform];
+            // Each cell needs to display the comparison by default, on hover displays further details
+            column.Cell = props => <Tooltip title={<div >
+                Raw Test Score: {this.handleProp(props.value, 'testScore')} <br/> 
+                Test CI: {this.handleProp(props.value, 'testCI')}  <br/>
+                Test JDK: {this.handleProp(props.value, 'testJdk')}	<br/> 
+                Test Sdk Resource: {this.handleProp(props.value, 'testSdkResource')}	<br/> 
+                Raw Baseline Score: {this.handleProp(props.value, 'baselineScore')} <br/> 
+                Basline CI: {this.handleProp(props.value, 'baselineCI')}  <br/>
+                Baseline JDK: {this.handleProp(props.value, 'baselineJdk')} <br/>
+                Baseline Sdk Resource: {this.handleProp(props.value, 'baselineSdkResource')}
+                </div> }>
+                <span onClick={() => this.handleLink(this.handleProp(props.value, 'buildUrl'))}> {this.handleProp(props.value, 'compare')} </span></Tooltip>;
+            column.getProps = (state, rowInfo) => (this.handleRegression(this.handleProp(rowInfo.row[platform], 'compare')));
+            newArray.push(column);
+        }
+        this.setState({columns:newArray, originalColumns: newArray});
+    }
+    // Set cell color based on comparison value
+    handleRegression (val) {
+        let color;
+        if (val <= 90) {
+            color = '#cc3232';}
+        else if (val <= 98) {
+            color =  '#F0F755';}
+        else if (val === 'N/A') {
+            color = 'grey';}
+        else if (val > 200) {
```

Comment:
Different color is fine but we shouldn't show the actual value for it since it's going to be confusing since we'll be showing the relative comparison percent for most of the cells.

The relative comparison value could be more than 200% in some rare cases such as converters and DAA tests so those cells should still be considered green.